test/features/data_usability_rating.feature

```diff
@@ -0,0 +1,55 @@
+@data_usability_rating
+Feature: Data usability rating
+
+    Scenario: As an admin user of my organisation, when I create a dataset with a HTML resource, I can verify the score is 0
+        Given "TestOrgAdmin" as the persona
+        When I log in
+        And I create a dataset with license "other-open" and "HTML" resource file "html_resource.html"
+        Then I wait for 10 seconds
+        When I reload
+        Then I should see an element with xpath "//h2[contains(text(), 'Openness')]"
```

Comment:
missing the verify score part, still good progress, all others are missing that step too.

Reply:
Should be sorted now.